core/src/components/accordion/accordion.tsx

```diff
@@ -14,7 +14,8 @@ const enum AccordionState {
 }
 
 /**
- * @virtualProp {"ios" | "md"} mode - The mode determines which platform styles to use.
+ * @virtualProp {"ios" | "md"} mode - The mode determines the platform behaviors of components.
```

Comment:
Nit: can we change this to 

```suggestion
 * @virtualProp {"ios" | "md"} mode - The mode determines the platform behaviors of the component.
```

to match the theme description